perfrunbook/utilities/install_perfrunbook_dependencies.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+install_al2023_dependencies () {
+  echo "------ INSTALLING UTILITIES ------"
+  yum install -y -q vim unzip git
```

Comment:
Its a nit, but please use the `dnf` command instead of `yum` since yum is deprecated now.